### ISSUE TITLE
Android app freezes and crashed when use FlutterPcmSound.release(). The same when back then access again to Screen play pcm.

### ISSUE THREAD

**Issue by BinhT1** [chipweinberger/flutter_pcm_sound#3]
i pull code from source and run example in android device.
i write a simple homepage and navigate to example PcmSoundApp page.
i access PcmSoundApp page and play music then stop.
i back to homepage and access again to PcmSoundApp page, then my emulator is freezes and crashed, the same when i use FlutterPcmSound.release()

here is Error

```dart
I/flutter ( 9655): [PCM] invoke: release
D/AudioTrack( 9655): stop(28): called with 283045 frames delivered
I/example.exampl( 9655): Waiting for a blocking GC ProfileSaver
I/example.exampl( 9655): WaitForGcToComplete blocked ProfileSaver on HeapTrim for 8.980ms
I/example.exampl( 9655): Thread[3,tid=9667,WaitingInMainSignalCatcherLoop,Thread*=0xe4208c10,peer=0x13100228,"SignalCatcher"]: reacting to signal 3
I/example.exampl( 9655):
I/example.exampl( 9655): Wrote stack traces to tombstoned
Lost connection to device.
Exited (sigterm)
```

**Comment by chipweinberger** [comment]
please open a PR to fix it. thanks.

**Comment by BinhT1** [comment]
i opened a pull request for this issue. This is the first time i have made a PR, please understand

**Comment by chipweinberger** [comment]
hey really awesome of you! thats something to be celebrated!

**Comment by chipweinberger** [comment]
fixed in 1.1.0
